app/models/feature_toggle.rb

```
@@ -4,4 +4,7 @@ def FeatureToggle.get_all_uhero
     FeatureToggle.where(%q{feature_toggles.universe = 'UHERO'})
   end
 
+  def FeatureToggle.is_set(name, universe = 'UHERO', default = false)
+    FeatureToggle.find_by(name: name, universe: universe).status rescue default
```

This fails silently, which could make debugging painful.